core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -82,6 +76,46 @@ def start_navigator(self) -> None:
     def start_serial(self, device: str) -> None:
         self.start_mavlink_manager(Endpoint("Master", self.settings.app_name, "serial", device, 115200, protected=True))
 
+    @staticmethod
+    def get_current_arch() -> str:
+        """Maps paltform.machine() outputs to docker architectures"""
+        arch_map = {"x86_64": "amd64", "armv7l": "arm"}  # TODO: differentiate arm64v8/ arm32v7
+        local_arch = platform.machine()
+        arch = arch_map.get(local_arch, None)
```

Comment:
Sorry if I was not clear, but what I'm saying is that:
if matches X86 is X86_64, otherwise it should be arm, the dict creates an unnecessary complex logic. 
We don't need to check each type of arm, any arm will run armv6 and since we only supports armv7, everything else that is not armv6 should run armv7.

Reply:
Don't agree. In your scenario, if it detects an ARM64 it will classify it as ARM and try to download the armv6/7 binary, which will not necessarily run on it.

Reply:
In fact this whole method came from PR #62, which was approved, and I copied it here in the intent to have common logic that will enter the common library on core in some time.
I would prefer that we try to keep the same method here and elaborate on the changes in it once we  have them on the final place, if that's possible.